samples/subsys/fs/littlefs/boards/stm32h747i_disco_m7.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 Christoph Heller
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &storage_partition;
+
+/ {
+	fstab {
+		compatible = "zephyr,fstab";
+		lfs1: lfs1 {
+			compatible = "zephyr,fstab,littlefs";
+			read-size = <512>;
+			prog-size = <512>;
+			cache-size = <512>;
+			lookahead-size = <4096>;
+			block-cycles = <512>;
+			partition = <&storage_partition>;
+			mount-point = "/lfsemmc";
+		};
+	};
+};
+
+&sdmmc1 {
+        sdmmc {
+		compatible = "zephyr,sdmmc-disk";
+		status = "okay";
```

Comment:
No need to enable the node. It's already enabled in board config

Reply:
OK